lib/src/arithmetic/addend_compressor.dart

```diff
@@ -170,8 +170,17 @@ class ColumnCompressor {
   /// The partial product array to be compressed
   final PartialProductArray pp;
 
+  /// The clk
+  Logic? clk;
+
+  /// Optional reset
+  Logic? reset;
+
+  /// Optional enable
+  Logic? enable;
+
   /// Initialize a ColumnCompressor for a set of partial products
```

Comment:
Worth some clarification on what the clock, reset, and enable are used for if they are present/not present.